openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -190,9 +190,10 @@ def add_field(
                 See `ALLOWED_RESTRICTIONS` for acceptable values.
 
             options (list): For "select" fields, a list of tuples
-                (value, display_name) representing the options available to
+                (value, display_name, default) representing the options available to
```

Comment:
We're not using this `default` parameter; please remove it. What we're doing is taking the normal `default` kwarg _for the field as a whole_; then, for each value we add to the select field, we're _marking_ it as a default if it matches the arg we received.